pkg/openstack/actuator.go

```diff
@@ -36,6 +36,7 @@ import (
 
 const (
 	rootOpenStackCredsSecretKey = "clouds.yaml"
+	rootOpenStackCACertKey      = "ca-bundle.pem"
```

Comment:
Is this really an OpenStack specific thing, or does the configmap contain CA cert data for any cloud environment using self-signed certificates?

Reply:
All public clouds don't use self-signed certificate. I don't have oVirt at hand to check where their certificate ends up.
https://docs.openshift.com/container-platform/4.5/installing/installing_rhv/installing-rhv-customizations.html#installation-launching-installer_installing-rhv-customizations